.gitignore

```diff
@@ -1,6 +1,26 @@
+# Bazel
 bazel-*
+
+# Maven (examples)
 target
-TAGS
+
+# IntelliJ IDEA
+.idea
+*.iml
+
+# Eclipse
 .classpath
 .project
 .settings
+.gitignore
```

Comment:
I don't think we should ignore the .gitignore file while we store it in the main repository.